tests/unit/test_req.py

```diff
@@ -636,3 +636,25 @@ def test_mismatched_versions(caplog, tmpdir):
         'Requested simplewheel==2.0, '
         'but installing version 1.0'
     )
+
+
+class TestRequirementTracker(object):
+    @pytest.fixture(autouse=True)
+    def blank_env(self):
+        with patch.dict(os.environ, clear=True):
+            yield
+
+    def test_no_side_effects_on_init(self):
```

Comment:
I would name this something starting with `test_init` (to group tests by method being tested) and be more specific about what you're actually testing, e.g. `test_init__does_not_set_PIP_REQ_TRACKER`.